src/safe_apps/tests/test_signals_ff_hook_events.py

```diff
@@ -131,24 +131,13 @@ def test_on_provider_create_with_safe_app(self) -> None:
 
         # Safe App Creation, Safe App Update
         assert len(responses.calls) == 2
-        assert isinstance(responses.calls[0], responses.Call)
         assert isinstance(responses.calls[1], responses.Call)
-        assert responses.calls[
-            0
-        ].request.body == f'{{"type": "SAFE_APPS_UPDATE", "chainId": "{chain_id}"}}'.encode(
-            "utf-8"
-        )
```

Comment:
I've removed these assertions because they are covered by other tests (i.e.: the creation step is already covered by other tests)